ipynb/android/workloads/Android_YouTube.ipynb

```diff
@@ -192,26 +194,36 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "2016-04-12 15:20:10,656 INFO    :         Target - Using base path: /home/pippo/work/lisa\n",
-      "2016-04-12 15:20:10,657 INFO    :         Target - Loading custom (inline) target configuration\n",
-      "2016-04-12 15:20:10,658 INFO    :         Target - Loading custom (inline) test configuration\n",
-      "2016-04-12 15:20:10,659 INFO    :         Target - Devlib modules to load: ['bl', 'cpufreq']\n",
-      "2016-04-12 15:20:10,660 INFO    :         Target - Connecting Android target [DEFAULT]\n",
-      "2016-04-12 15:20:11,142 INFO    :         Target - Initializing target workdir:\n",
-      "2016-04-12 15:20:11,144 INFO    :         Target -    /data/local/tmp/devlib-target\n",
-      "2016-04-12 15:20:13,757 INFO    :         Target - Topology:\n",
-      "2016-04-12 15:20:13,759 INFO    :         Target -    [[0, 1, 2, 3], [4, 5]]\n",
-      "2016-04-12 15:20:14,399 INFO    :         FTrace - Enabled tracepoints:\n",
-      "2016-04-12 15:20:14,400 INFO    :         FTrace -   sched_switch\n",
-      "2016-04-12 15:20:14,401 INFO    :         FTrace -   sched_load_avg_cpu\n",
-      "2016-04-12 15:20:14,402 INFO    :         FTrace -   cpu_frequency\n",
-      "2016-04-12 15:20:14,402 INFO    :         FTrace -   cpu_capacity\n",
-      "2016-04-12 15:20:14,403 WARNING :        TestEnv - Wipe previous contents of the results folder:\n",
-      "2016-04-12 15:20:14,404 WARNING :        TestEnv -    /home/pippo/work/lisa/results/Android_Youtube\n",
-      "2016-04-12 15:20:14,434 INFO    :        TestEnv - Set results folder to:\n",
-      "2016-04-12 15:20:14,435 INFO    :        TestEnv -    /home/pippo/work/lisa/results/Android_Youtube\n",
-      "2016-04-12 15:20:14,435 INFO    :        TestEnv - Experiment results available also in:\n",
-      "2016-04-12 15:20:14,435 INFO    :        TestEnv -    /home/pippo/work/lisa/results_latest\n"
+      "2016-12-06 19:46:11,262 INFO    : TestEnv      : Using base path: /user_projects/lisa/lisa/lisa\n",
```

Comment:
Can we normalize the "anonimized" path to be something more generic... maybe we can assume a default installation in Vagrant, i.e. `/home/lisa` ?